app/helpers/application_helper.rb

```diff
@@ -32,7 +32,7 @@ def policy_description(policy)
   end
 
   def feedback_url
-    current_policy.feedback_url
+    "https://docs.google.com/forms/d/e/1FAIpQLSfwPUxmNHqSnQ6RJ-0nedu5F2FRibBF5UIJ_EciTPcWQg581A/viewform"
```

Comment:
I just tried this link and it goes to a form titled "Claim a payment for teaching maths or physics - feedback form". Are we sure that this is the right link?

Reply:
I've asked in Slack